src/erlcloud_sns.erl

```diff
@@ -286,8 +288,8 @@ list_endpoints_by_platform_application(PlatformApplicationArn, NextToken, Access
     list_endpoints_by_platform_application(PlatformApplicationArn, NextToken, new_config(AccessKeyID, SecretAccessKey)).
```

Comment:
Line 288 is too long:     list_endpoints_by_platform_application(PlatformApplicationArn, NextToken, new_config(AccessKeyID, SecretAccessKey))..